libraries/Microsoft.Bot.Connector/Teams/TeamsOperationsExtensions.cs

```diff
@@ -143,5 +145,214 @@ public static async Task<MeetingNotificationResponse> SendMeetingNotificationAsy
                 throw new InvalidOperationException("TeamsOperations with SendMeetingNotificationWithHttpMessagesAsync is required for SendMeetingNotificationAsync.");
             }
         }
+
+        /// <summary>
+        /// Sends a message to a list of Teams users.
+        /// </summary>
+        /// <param name='operations'>
+        /// The operations group for this extension method.
+        /// </param>
+        /// <param name='activity'>
+        /// The activity to send.
+        /// </param>
+        /// <param name='teamsMembers'>
+        /// The list of members recipients for the message.
+        /// </param>
+        /// <param name='tenantId'>
+        /// The tenant ID.
+        /// </param>
+        /// <param name='cancellationToken'>
+        /// The cancellation token.
+        /// </param>
+        /// <returns>The operation Id.</returns>
+        public static async Task<string> SendMessageToListOfUsersAsync(this ITeamsOperations operations, IActivity activity, List<object> teamsMembers, string tenantId, CancellationToken cancellationToken = default(CancellationToken))
+        {
+            if (operations is TeamsOperations teamsOperations)
+            {
+                using (var result = await teamsOperations.SendMessageToListOfUsersAsync(activity, teamsMembers, tenantId, null, cancellationToken).ConfigureAwait(false))
```

Comment:
I'm not fully familiar with the implementation here, and I'm wondering why one of the parameters is always null, when we're calling methods from teamsOperations?

Reply:
The null parameter is for _customHeaders_ that are used for creating the request. We removed the null value in the calls because it defaults to null.